packages/esm-appointments-app/src/appointments/forms/create-edit-form/appointments-form.component.tsx

```diff
@@ -217,7 +218,7 @@ const AppointmentForm: React.FC<AppointmentFormProps> = ({ appointment, patientU
           dateFormat="d/m/Y"
           datePickerType="single"
           id="visitDate"
-          minDate={patientAppointment.visitDate}
+          minDate={today}
```

Comment:
Note that I set the minDate to be "today", which seems reasnable for a POC scheduling functionality.  I'm fine with removing any constraint if people think that is better to allow retrospectively entering/scheduling appointments after they already should have happened.